Vagrantfile

```diff
@@ -66,5 +66,5 @@ Vagrant.configure(2) do |config|
   # Enable provisioning with a shell script. Additional provisioners such as
   # Puppet, Chef, Ansible, Salt, and Docker are also available. Please see the
   # documentation for more information about their specific syntax and use.
-  config.vm.provision :shell, :path => "scripts/install.sh"
+  config.vm.provision :shell, :path => "v1/scripts/install.sh"
```

Comment:
This I will remove too, as suggested by @abishekvashok in earlier PR. Changed the path to run locally using Vagrant.